templates/telegraf.conf.epp

```diff
@@ -68,3 +68,31 @@
 <% } -%>
 <% } -%>
 <%# -%>
+
```

Comment:
In #20 I rework this file to include parameter validation and suggest doing so here too. That would mean adding an entry for `postgres_host` to the beginning of it.